src/webWorker/dataLoader.worker.js

```diff
@@ -3,11 +3,3 @@ import decodeAndTransformTask from './decodeAndTransformTask.js'
 
 // register our task
 registerTaskHandler(decodeAndTransformTask)
-
```

Comment:
These changes are related to registering the tasks using the newer Worker registration format.